server/src/test/java/org/apache/cassandra/sidecar/server/ServerTest.java

```diff
@@ -308,6 +308,16 @@ void unrecognizedAuthenticationHandlerSet()
                               "UnrecognizedAuthenticationHandler has not been registered");
     }
 
+    @Test
+    @DisplayName("Invalid access control config, unrecognized authorization provider set")
+    void unrecognizedAuthorizationProviderSet()
+    {
+        assertThatThrownBy(() -> configureServer("config/sidecar_unrecognized_authorizer.yaml"))
+        .hasCauseInstanceOf(RuntimeException.class)
```

Comment:
can it throw a dedicated exception instead of `RuntimeException`?

Reply:
It throws dedicated exception `ConfigurationException` I was by mistake checking for `RuntimeException` in test. Updated it